cns/restserver/ipampoolmonitor.go

```diff
@@ -0,0 +1,138 @@
+package restserver
+
```

Comment:
For now, it is fine to keep the PoolMonitor under RestServer, but ideally this should be outside restserver package.